src/app/components/applications/SSV/MyAccount/components/EditOperatorDetails/InputFieldComponent.tsx

```diff
@@ -14,7 +14,8 @@ const InputFieldComponent = ({ fieldKey, extendClass, placeholder  }: { fieldKey
 
     const onChangeHandler = (event: any) => {
         const { value } = event.target;
-        setCurrentValue(metadataStore.setMetadataValue(fieldKey, value.trim()));
+        setCurrentValue(value.length > 1 ? value : value.trim());
```

Comment:
This is weird condition, lets do simply `value.trim()` instead, it will work for both cases